doc/whats_new.rst

```diff
@@ -29,6 +29,7 @@ Bug
 
 - fix bug in :func:`write_raw_bids` where raw.info['subject_info'] can be None, by `Adam Li`_ (`#392 <https://github.com/mne-tools/mne-bids/pull/392>`_)
 - :func:`read_raw_bids` will now read all channels from electrodes.tsv. Channels with coordinates 'n/a' will also be included but their location set to `np.nan` in the `raw` object, by `Adam Li`_ (`#393 <https://github.com/mne-tools/mne-bids/pull/393>`_)
+- :func:`read_raw_bids` will read in raw even if participants.tsv does not match mne-python standards, by `Adam  Li`_ (`#397 <https://github.com/mne-tools/mne-bids/pull/397>`_)
```

Comment:
Since you fixed this bug in the same release cycle, there is no need to add it here.